### PR TITLE
change text on left sidebar

### LINKED ISSUE #1936: Change text on left sidebar
Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you :gift_heart:

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!
:thinking: What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
#### Please describe the problem

Change the "View all" to "Contributors for the topic" on the left sidebar on the wiki pages.

:clipboard: Step by Step

:raising_hand_woman: Claim this issue: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

:memo: Change the text in the following lines
https://github.com/publiclab/plots2/blob/9e0f5e6ba58b72390cb7772c9ebbb088d672d4a5/app/views/sidebar/_related.html.erb#L51-L51
and
https://github.com/publiclab/plots2/blob/9e0f5e6ba58b72390cb7772c9ebbb088d672d4a5/app/views/sidebar/_related.html.erb#L62

:floppy_disk: Commit your changes

:twisted_rightwards_arrows: Start a Pull Request.

:checkered_flag: Done Ask in comments for a review :)

:thinking::question: Questions?

Leave a comment below!

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. If someone seem stuck, offer them some help! Otherwise, take a look at some other issues you can help with. Thanks!
Thank you!

Your help makes Public Lab better! We deeply appreciate your helping refine and improve this site.

### PR DESCRIPTION
**Pull request by ankitbhardwaj78**
Fixes #1936 

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by SidharthBansal** [comment]
@ankitbhardwaj78 as per suggestion in https://github.com/publiclab/plots2/issues/1936#issuecomment-356661149 by @jywarren, please change "the" to "this".
Thanks for the PR

**Comment by ankitbhardwaj78** [comment]
fixex #1936

**Comment by jywarren** [comment]
Hi, thanks! I've retitled it so it's easier to see the purpose in the list of pull requests. 

Do you think you could un-add the Gemfile.lock? Just reverting it to the original and adding that as an additional commit should cancel out the changes. We're just not sure this Gemfile will compile on production and it's unrelated to your fixes. Thank you very much!

**Comment by ankitbhardwaj78** [comment]
@jywarren thanks for your suggestions.I have changed it as you said can you please take a review again and suggest any changes if you want as i am new to this

**Comment by ankitbhardwaj78** [comment]
@jywarren this is ready please take a review

File: app/views/sidebar/_related.html.erb [hunk 1]
```diff
@@ -59,7 +59,7 @@
         <a href="/profile/<%= author.name %>"><%= author.name %></a>
       </p>
       <% end %>
-      <p><a href="/contributors/<%= params[:id] %>"><i class="fa fa-list"></i> <%= t('sidebar._related.view_all') %> &raquo;</a></p>
+      <p><a href="/contributors/<%= params[:id] %>"><i class="fa fa-list"></i> <%= t('sidebar._related.contributors_for_this_topic') %> &raquo;</a></p>
```
Comment (jywarren) [inline app/views/sidebar/_related.html.erb]: This looks good but we need a matching locale file -- that'll provide the text here, like in this example:

https://github.com/publiclab/plots2/blob/8e1a7a02091781a5cf572236befeb79bc5292e35/config/locales/views/sidebar/_related/en.yml

Make sense? Thank you!!!

**Comment by ankitbhardwaj78** [comment]
@jywarren  please take a review

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -245,11 +244,25 @@
   });
 
   $('#link_social_media').click(function(){
+<<<<<<< HEAD
```
Comment (jywarren) [inline app/views/users/profile.html.erb]: Oops, looks like a merge is incomplete, take a look! But it's a quick fix :-)

**Comment by jywarren** [comment]
Also I think you committed an sqlite file -- but otherwise this is looking good! Almost there, thanks!!!

**Comment by ankitbhardwaj78** [comment]
@jywarren this is complete now

**Comment by jywarren** [comment]
Super!!! 🎉

**Pull request merged**